project/profiles/capacity4more/modules/c4m/og/c4m_og/plugins/entityreference/selection/C4MOgSelectionHandler.class.php

```diff
@@ -140,7 +140,7 @@ public function buildEntityFieldQuery(
       $target_access = og_user_access($group_type, $group['gid'], "create $node_type content");
       // Any member can edit a wiki page unless a power user has changed it
       // specifically for a specific node.
-      if (!empty($this->entity->nid) && $node_type == 'wiki_page') {
```

Comment:
It fixed the bug we had, see travis for https://github.com/capacity4dev/capacity4more/pull/1482/commits/84154e0552dd5d8c4bef571a6f1b876071472f36